src/Microsoft.ML.CpuMath/Avx.cs

```diff
@@ -1,1165 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
nit: Removing dead code seems unrelated to the PR.

Reply:
Some of the functions had the similar names like matmulPX in avx.cpp files which were not getting used anywhere so in order to avoid confusion I removed all the deadCode. Earlier I was delaying it until the aligned array removal PR.